### ISSUE TITLE
ComboTrackSelector fails on JBrowse 1.13.1-alpha.0.

### ISSUE THREAD

**Issue by thomasvangurp** [Arabidopsis-Information-Portal/ComboTrackSelector#1]
It seems as if the following code in main.js is causing an issue by trying to "clone" a config which leads to an error:
```javascript
        // load all the classes we need
        require( [ tl_class ],
                 lang.hitch( this, function( trackListClass ) {
                     // instantiate the tracklist and the track metadata object
                     this.trackListView = new trackListClass(
                         lang.mixin(
                             lang.clone( newconfig.trackSelector ) || {},
                             {
                                 trackConfigs: newconfig.tracks,
                                 browser: this,
                                 trackMetaData: this.trackMetaDataStore
                             }
                         )
                     );
```

<img width="940" alt="solynta_jbrowse_chr02_12401705__12401861" src="https://user-images.githubusercontent.com/1284722/38198955-182cb8e4-368f-11e8-9ac8-935c372a475c.png">

**Comment by vivekkrish** [comment]
Hi @thomasvangurp,

Thank you for reporting this error! I tested the latest JBrowse release 1.13.1 and noticed that ComboTrackSelector seems to be working well. Here are the steps I took to test:
1. Download latest JBrowse-1.13.1-dev and run `setup.sh`
```
$ cd /path/to/htdocs
$ git clone https://github.com/GMOD/jbrowse.git jbrowse-dev
$ cd jbrowse-dev
$ ./setup.sh
```
2. Download latest ComboTrackSelector and run `npm run build`:
```
$ cd plugins
$ git clone https://github.com/Arabidopsis-Information-Portal/ComboTrackSelector.git
$ cd ../
$ npm run build
```
3. Set up a test JBrowse dataset which uses the ComboTrackSelector. I set this up based on the demo configs on the JBrowse Registry (https://github.com/GMOD/jbrowse-registry/tree/master/demos/combotrackselector):
```
$ ls -R /path/to/htdocs/jbrowse-dev/combotrackselector
combotrackselector/:
epiccogeMetaData.csv  seq  trackList.json  trackList2.json  tracks.conf

combotrackselector/seq:
refSeqs.json
```
4. Opened JBrowse to this dataset: http://localhost/jbrowse-dev/?data=combotrackselector
![image](https://user-images.githubusercontent.com/711637/38526608-0a8384c0-3c25-11e8-84ce-4e995ab3c212.png)

Based on the above steps, I was able to get a working version of the plugin.

On your end, could you check to see if you ran `npm run build` as specified in Step 2 above?

Sorry for the inconvenience caused.

**Comment by thomasvangurp** [comment]
Managed to get it to work, although the layout looks a bit messed up in JBrowse-1.14.0-alpha.0
<img width="265" alt="solynta_jbrowse_chr01_20866982__20867171" src="https://user-images.githubusercontent.com/1284722/38930759-3e63ba60-4311-11e8-961a-f3ef012d079f.png">. Any idea on how to fix this?

**Comment by vivekkrish** [comment]
Hi @thomasvangurp, a minor fix #3 was recently merged to the main branch. Could you please test this latest version to see if it resolves the rendering issue? Thanks!

**Comment by thomasvangurp** [comment]
Hi @vivekkrish I've replicated the exact steps that you explained above, still I am getting this:
<img width="529" alt="jbrowse_1_12170743__18256277" src="https://user-images.githubusercontent.com/1284722/42224060-7edc829e-7ed9-11e8-889b-728e2077a3da.png">.

 So, #2 is still open I'm afraid...

**Comment by thomasvangurp** [comment]
any updates on this?

**Comment by vivekkrish** [comment]
Hi @thomasvangurp, please go ahead and use the latest plugin from the `master` branch. The issue should've been fixed, thanks to the most recent PRs, #4 and #5.

Thank you!
